1.0.0 changes

cs-abhishek-shukla opened the pull request:
Added annotated screenshots
Created step by step instructions

cs-abhishek-shukla committed:
1.0.0 changes
Added annotated screenshots
Created step by step instructions

cs-suyog-jinturkar commented on README.md:
```diff
@@ -8,16 +8,16 @@
 The Submit Content Form Widget is a user-friendly interface designed for community members to share their solutions. 
-The Submit Content Form Widget is part of the larger Submit Your Content Solution pack, which includes associated playbooks for email. This integration streamlines the submission process and ensures efficient handling of submitted solutions.
+
+The Submit Content Form Widget is part of the larger Submit Your Content Solution pack that empowers community members to seamlessly submit their solutions on FortiSOAR’s Content Hub. By launching their FortiSOAR instance and leveraging the solution pack's playbooks, users can streamline the submission process, ensuring efficient handling of their valuable contributions.
```
Need more information on why user should use this widget instead of submitting it via email. We can mention like consistent user experience within FSR only. With predefined steps, forms, etc. makes is very easy to use solution for all our community users.

cs-suyog-jinturkar commented on docs/setup.md:
```diff
@@ -5,14 +5,15 @@
 # Configuration
 
-[This] (https://github.com/fortinet-fortisoar/solution-pack-community-submissions/blob/develop/doc/setup.md#configuration) guide provides an approach to choose the medium to send email. Eg. using exchange or the normal SMTP route.
+The widget is a part of **Submit Your Content** solution pack and does not require any standalone configuration.
```
This is pre-req. Pleae highlight this here as well in overview page.

cs-suyog-jinturkar commented on docs/usage.md:
```diff
@@ -3,15 +3,61 @@
+
+    - **Upload a custom file** - Upload a `tgz` file (for connectors and widgets) or a `zip` file (for solution packs) of your content developed in a separate environment.
+
+10. Click **Submit Content** button to complete the submission process.
```
What happenes in the background is missing here and its a very important step.

cs-suyog-jinturkar replied:
You can also highlight Content submit page and info mentioned in the texts

cs-suyog-jinturkar reviewed (commented):
Added review comments

cs-yash-patodkar commented on README.md:
```diff
@@ -8,16 +8,16 @@
 
 ## Next Steps
 
 | [Installation](./docs/setup.md#installation) | [Configuration](./docs/setup.md#configuration) | [Usage](./docs/usage.md) | [Contents](./docs/contents.md) |
```
Content link is not working

cs-abhishek-shukla committed:
1.0.0 changes
- Annotated screenshots
- moved content to SP
- Gave a brief idea about the widget
- added links to SP
